stacks/java-wildfly/stack.yaml

```diff
@@ -0,0 +1,9 @@
+name: ava-wildfly
+description: 'JakartaEE  application using WildFly'
+displayName: WildFly
+icon: https://design.jboss.org/wildfly/logo/final/wildfly_logomark.svg
+versions:
+  - version: 1.1.0
+  - version: 2.0.0
+    default: true
```

Comment:
I'm not too familiar with this stack so just wondering if there's any concerns / potential problems with defaulting to the new version

Reply:
We want to default devfile 2.2 for stacks as part of https://github.com/devfile/api/issues/1129 so this change lines up with that initiative.